vendor/assets/stylesheets/rolodex/settings/variables/_typography.scss

```diff
@@ -23,67 +23,12 @@ $font-family-open-sans: "Open Sans", Verdana, Helvetica, sans-serif;
 $font-family-helvetica: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
 $font-family-monospace: "Source Code Pro", Inconsolata, "Lucida Console", Terminal, "Courier New", Courier;
 
-// Weights
-
-$font-weight-book:   300;
```

Comment:
Should we keep these?  Seems like they are handy for not having to remember exactly which values we use.